### DISCUSSION TITLE
github account

### DISCUSSION THREAD

**Discussion by Ngocgiau2000** [community/community#75950] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

After restoring my phone, I forgot my github gmail account, now how to get that github account back, please help me, thank you

**Comment by peterroth** [comment]
How did you post this message? If this is the account you need, go to settings and checkfor the registered e-mail address.

**Comment by Ngocgiau2000** [comment]
Thank you because it's another account of mine. not an existing github
account

Vào 12:41 SA, CN, 19 thg 11, 2023 Péter Róth ***@***.***> đã
viết:
